recipe/build.sh

```diff
@@ -88,6 +88,12 @@ else
   export LDFLAGS="${LDFLAGS} -lrt"
 fi
 
+ARCH=$(uname -m)
+if [ ${ARCH} == "aarch64" ]; then
+    NVARCH=sbsa
+else
+    NVARCH=${ARCH}
+fi
```

Comment:
This should be inside the `${cuda_compiler_version} != "None"` statement since it likely doesn't apply well for osx.

You can also likely just use `target-architechture` to know the TARGET, instead of the host.

Reply:
sorry, i think it is `target_platform` see the osx cross compilation lines.